### PR TITLE
add two books in `-zh` list in `python` section

### PR DESCRIPTION
**Pull request by zhongjiajie**
add two books in `-zh` list in `python` section.

### CODE REVIEW

File: free-programming-books-zh.md [hunk 1]
```diff
@@ -613,10 +613,12 @@
 * [Python 3 文档(简体中文) 3.2.2 documentation](http://docspy3zh.readthedocs.org/en/latest/)
 * [Python Cookbook第三版](http://python3-cookbook.readthedocs.io/zh_CN/latest/) (作者：David Beazley, Brian K.Jones 翻译：熊能)
 * [Python 中文学习大本营](http://www.pythondoc.com)
+* [Python之旅](https://funhacks.net/explore-python) (作者：Ethan)
```
Comment (eshellman) [inline free-programming-books-zh.md]: nice resource, but it's not HTTPS. use HTTP.

File: free-programming-books-zh.md [hunk 2]
```diff
@@ -613,10 +613,12 @@
...
 * [Python教程 - 廖雪峰的官方网站](http://www.liaoxuefeng.com/wiki/0014316089557264a6b348958f449949df42a6d3a2e542c000)
 * [像计算机科学家一样思考Python](https://www.ctolib.com/docs/sfile/think-python-2e/0.html) (中英对照版 作者：Allen B. Downey 翻译：大胖哥)
 * [深入 Python 3](https://github.com/jiechic/diveintopython3)
 * [笨办法学 Python](http://old.sebug.net/paper/books/LearnPythonTheHardWay/)
+* [简明 Python 教程](https://bop.molun.net) (作者：Swaroop C H 译者：沈洁元、漠伦)
```
Comment (eshellman) [inline free-programming-books-zh.md]: This is really great!

**Comment by eshellman** [comment]
Thank you @zhongjiajie

**Comment by zhongjiajie** [comment]
@eshellman Thank you to modify the link

**Pull request merged**